src/bindings/python/PyBuiltinTransform.cpp

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#include "PyTransform.h"
+
+namespace OCIO_NAMESPACE
+{
+
+void bindPyBuiltinTransform(py::module & m)
+{
+    py::class_<BuiltinTransform, 
+               BuiltinTransformRcPtr /* holder */, 
+               Transform /* base */>(m, "BuiltinTransform")
+        .def(py::init(&BuiltinTransform::Create))
```

Comment:
Can be done in a later PR, but would be good to have a convenience constructor like the other transforms.

Reply:
Good point. In a Later PR.